app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java

```diff
@@ -145,6 +157,19 @@ public void displayDescriptionActivityData() {
             recButton.setEnabled(false);
             findViewById(R.id.activity_description_stats_layout).setVisibility(View.GONE);
             findViewById(R.id.activity_description_stats_data_layout).setVisibility(View.GONE);
+           }
+        } else {
+            if (activity.getParticipantId().size() < activity.getNumberParticipant()) {
+                findViewById(R.id.activityRegisterDescription).setVisibility(View.VISIBLE);
+                findViewById(R.id.activityGPSRecDescription).setVisibility(View.GONE);
+                findViewById(R.id.activityChatDescription).setVisibility(View.GONE);
+            } else {
+                findViewById(R.id.activityRegisterDescription).setVisibility(View.VISIBLE);
+                findViewById(R.id.activityRegisterDescription).setEnabled(false);
+                ((TextView) findViewById(R.id.activityRegisterDescription)).setText("No more free places");
+                findViewById(R.id.activityGPSRecDescription).setVisibility(View.GONE);
+                findViewById(R.id.activityChatDescription).setVisibility(View.GONE);
+            }
```

Comment:
Why do you use GONE instead of INVISIBLE here? Is it a question of visual aspect and distribution of the buttons on the ActivityDescriptionActivity page?